file.go

```diff
@@ -126,9 +157,10 @@ func (f *File) GetSection(name string) (*Section, error) {
 	}
 
 	sec := f.sections[name]
-	if sec == nil {
+	if sec == nil || len(sec) == 0 {
 		return nil, fmt.Errorf("section '%s' does not exist", name)
```

Comment:
Would be nice have this fixed altogether in this PR:
```suggestion
		return nil, fmt.Errorf("section %q does not exist", name)
```